### DISCUSSION TITLE
Github classroom assignment is due by Jun 30, 2023, 23:30 IST

### DISCUSSION THREAD

**Discussion by sivapullaiahmangali** [community/community#45256] [GitHub Education]
can any one help me to do this assignment is due by Jun 30, 2023, 23:30 IST
i am new to github

**Comment by jge162** [comment]
What assignment are u doing?

**Comment by twisted-nematic57** [comment]
I can't do your homework but I can certainly help you use Git. But you have to remember that most Git problems can be solved by searching on the Internet.
